packages/grid/x-data-grid/src/hooks/features/filter/gridFilterUtils.ts

```diff
@@ -22,25 +22,21 @@ import {
   gridVisibleColumnFieldsSelector,
 } from '../columns';
 
-// Fixes https://github.com/mui/mui-x/issues/10056
-const globalScope = (typeof window === 'undefined' ? globalThis : window) as any;
-const evalCode = globalScope[atob('ZXZhbA==')] as <T>(source: string) => T;
```

Comment:
We're not executing in sloppy mode, we're already executing in strict mode due to `eval` being stored in a variable with a different name.

Reply:
True